core/controllers/classifier_test.py

```diff
@@ -168,30 +168,31 @@ def setUp(self):
             secret, self.payload['message'])
 
     def test_next_job_handler(self):
-        json_response = self.post_json('/ml/nextjobhandler',
-                                       self.payload, expect_errors=False,
-                                       expected_status_int=200)
+        json_response = self.post_json(
+            '/ml/nextjobhandler', self.payload, expect_errors=False)
```

Comment:
Why are we getting rid of expected_status_int, here and below?

Reply:
Oh, my bad. When I changed POST to GET I removed them and forgot to add them back. I will add them.